### PR TITLE
Add atreus, ergotravel and org60 keymaps

### PR DESCRIPTION
**Pull request by JarredSteenvoorden**
## Description

Add my atreus, ergotravel and org60 keymaps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/atreus62/keymaps/jarred/config.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```
Comment (mechmerlin) [inline keyboards/atreus62/keymaps/jarred/config.h]: You can `#pragma once` this include guard

File: keyboards/ergotravel/keymaps/jarred/config.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```
Comment (mechmerlin) [inline keyboards/ergotravel/keymaps/jarred/config.h]: You can also `#pragma once` this include guard.

File: keyboards/atreus62/keymaps/jarred/config.h (hunk 1 above)
Comment (drashna) [inline keyboards/atreus62/keymaps/jarred/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```

File: keyboards/atreus62/keymaps/jarred/config.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#define CONFIG_USER_H
```
Comment (drashna) [inline keyboards/atreus62/keymaps/jarred/config.h]:
```suggestion
```

File: keyboards/atreus62/keymaps/jarred/config.h [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+#include "config_common.h"
+
+// My hand wire diodes are in the opposite direction to the Atreus62 PCB
+#undef DIODE_DIRECTION
+#define DIODE_DIRECTION COL2ROW
+
+#endif
```
Comment (drashna) [inline keyboards/atreus62/keymaps/jarred/config.h]:
```suggestion
```

File: keyboards/atreus62/keymaps/jarred/config.h [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/atreus62/keymaps/jarred/config.h]: This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant. 

```suggestion
```

File: keyboards/ergotravel/keymaps/jarred/config.h (hunk 2 above)
Comment (drashna) [inline keyboards/ergotravel/keymaps/jarred/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```

File: keyboards/ergotravel/keymaps/jarred/config.h [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+#define CONFIG_USER_H
```
Comment (drashna) [inline keyboards/ergotravel/keymaps/jarred/config.h]:
```suggestion
```

File: keyboards/ergotravel/keymaps/jarred/config.h [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+// #define USE_SERIAL
+#define USE_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#endif
```
Comment (drashna) [inline keyboards/ergotravel/keymaps/jarred/config.h]:
```suggestion
```

File: keyboards/ergotravel/keymaps/jarred/config.h [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/ergotravel/keymaps/jarred/config.h]: This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant. 

```suggestion
```

**Comment by JarredSteenvoorden** [comment]
Updated config files with suggested changes, thanks!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**